meta-balena-common/recipes-core/images/balena-image-boot-initramfs.bb

```diff
@@ -0,0 +1,22 @@
+DESCRIPTION = "This minimal image is executed by a vendor provided bootloader. \
+It contains all the BalenaOS bootloader specific features, and will kexec the \
+BalenaOS initramfs"
+
+PACKAGE_INSTALL = " \
+    kexecboot \
```

Comment:
kexecboot is removed further down - squashing it would make it easier on reviewers.